app/src/main/java/com/wizeline/academy/animations/ui/detail/DetailViewModel.kt

```diff
@@ -4,9 +4,11 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.wizeline.academy.animations.core.data.repository.LoreIpsumRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
 import javax.inject.Inject
 import kotlin.random.Random
 
+@HiltViewModel
```

Comment:
you include hiltViewModel but  remove @Inject from DetailFragment, did you have problems injecting the ViewModel?

Reply:
That's the way it works with HiltViewModels, accordingly to the documentation; you can check it out on the [official documentation ](https://dagger.dev/hilt/view-model.html)